src/avt/Filters/avtThresholdFilter.C

```diff
@@ -726,7 +726,7 @@ avtThresholdFilter::ThresholdOnRanges(vtkDataSet *in_ds,
 vtkDataSet *
 avtThresholdFilter::ThresholdToPointMesh(vtkDataSet *in_ds)
 {
-    
+
```

Comment:
Unnecessary whitespace only changes? Should be reverted.

Reply:
This actually isn't an added whitespace. What happened here is that there used to be a whitespace with several actual spaces in it (it looks like maybe 3 or 4 spaces in that solid red block). At some point, I deleted those 3 spaces without knowing it... So, the file still has the same whitespace as before, it's just that this revision now doesn't have the spaces in it. I can add them back, but it seemed a little odd to do that. Thoughts?

Reply:
I should have noticed that by the blobs on the left. My bad. Yes, it's a good idea to remove that type of whitespace. The first line in ThresholdToPointMesh doesn't need to be blank, though, it can be deleted altogether.

Reply:
Sounds good. I'll go ahead and remove that one.

Reply:
Done!